packages/backend/serverless-queue/src/validations/queue-job-validate-default.ts

```diff
@@ -0,0 +1,48 @@
+import Joi from "joi";
+import * as R from "ramda";
+import { PaginationParameterAttributes, getPaginationParameterSchema } from "@gravitywelluk/validation-utils";
+import { $enum } from "ts-enum-util";
+
+import { QueueJobAttributes, QueueJobStatus } from "../models/queue-job";
+
+export type QueueJobDefaultSchemaAttributes =
+  QueueJobAttributes &
+  PaginationParameterAttributes<QueueJobAttributes> &
+  QueueJobFilters;
+
+export const queueJobSchemaDefault = {
```

Comment:
How is this needed outside the module? I would assume the queue is always responsible for handling the creation jobs in the DB

Reply:
ive mainly put the validation in for pagination, yet to add this to the creation function as we pass the validation through for the job data as thats the main part to validate on creation.

Reply:
makes sense!